### DISCUSSION TITLE
1 Şubattan Sonra Ne Olacak ?

### DISCUSSION THREAD

**Discussion by Odysseusksk** [IBAX-io/go-ibax#1611] [Rewards]
Rütbelerin henüz 11 adedinde hiç bir kullanıcı olmadığını biliyoruz. En üst seviye şu an için "Corporal"  Boşuna getirmediler o kadar rütbeyi herhalde. Hepsi de puan sıralamasına göre. Demek ki Puan kazanmaya devam edeceğiz, demek ki puan kazanmamız için yeni görevler, yarışmalar gelecek. Bir başka ifadeyle IDO öncesi en az bir tur daha Airdrop olacak benim görüşüme göre.

Peki sizler neler düşünüyorsunuz ? 1 Şubattaki Airdroptan sonra neler bekliyor bizleri ?

**Comment by muco3327** [comment]
sanmam öyle olacağını. 1 olunca biter hepsi

**Comment by yusftkn1** [comment]
bence herkes pay alıcak
,

**Comment by Snglalgl** [comment]
Net açıklama neden yok gruplar da da herkes birbirine girmiş

**Comment by Odysseusksk** [comment]
Bugün yeni gelen 5000 puanlık RT görevi ile "Corporal" rütbesine sahip kullanıcı sayısı artarken, "Infantry" rütbeli kullanıcı aynı oranda azalacak. Dün 1 Şubattan sonra ne olacak diye sorarken bu durum yoktu henüz. Bir bakıma iddiam doğru çıkmaya başladı.

1 Şubat sonrası yeni bir airdrop gelebileceğini düşünüyorum

**Comment by gokhan85** [comment]
1 subattan sonra ananin ami olacak.cok mu merak ettin
